Routing/Loader/DirectoryRouteLoader.php

```diff
@@ -34,6 +37,10 @@ public function __construct(RestRouteProcessor $processor)
      */
     public function load($resource, $type = null)
     {
+        if (isset($resource[0]) && '@' === $resource[0]) {
+            $resource = $this->fileLocator->locate($resource);
```

Comment:
I mean removing the ``if``. That way we would have clear messages in case the folder does not exist and we would be able to support relative paths.